js/ascendex.js

```diff
@@ -1448,7 +1448,13 @@ module.exports = class ascendex extends Exchange {
             market = this.market (symbol);
             request['symbol'] = market['id'];
         }
-        const response = await this.v1PrivateAccountCategoryDeleteOrderAll (this.extend (request, params));
+        let method = 'v1PrivateAccountCategoryDeleteOrderAll';
+        if (market['swap']) {
```

Comment:
The market variable is not always defined. Hence, we cannot always dereference a `swap` key from it.